x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -134,25 +142,59 @@ export class ActionsClient {
       actionTypeId: result.attributes.actionTypeId as string,
       name: result.attributes.name as string,
       config: result.attributes.config as Record<string, any>,
+      isPreconfigured: false,
     };
   }
 
   /**
    * Get an action
    */
   public async get({ id }: { id: string }): Promise<ActionResult> {
+    const preconfiguredConnector = this.preconfiguredConnectors.find(
+      pConnector => pConnector.id === id
+    );
+    if (preconfiguredConnector !== undefined) {
+      return {
+        id,
+        actionTypeId: preconfiguredConnector.actionTypeId,
+        name: preconfiguredConnector.name,
+        config: preconfiguredConnector.config,
+        description: preconfiguredConnector.description,
+        isPreconfigured: true,
+      };
+    }
     const result = await this.savedObjectsClient.get<RawAction>('action', id);
 
     return {
       id,
       actionTypeId: result.attributes.actionTypeId,
       name: result.attributes.name,
       config: result.attributes.config,
+      isPreconfigured: false,
     };
   }
 
   /**
-   * Find actions
+   * Get all actions connectors with preconfigured connectors
+   */
+  public async getAll(): Promise<FindActionResult[]> {
+    const savedObjectsActions = (
+      await this.savedObjectsClient.find<RawAction>({
+        perPage: 10000,
```

Comment:
It's recommended to have numbers [stored in a variable](https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#magic-numbersstrings). You can copy (including comment) from what was done in the UI code here: https://github.com/elastic/kibana/blob/4efeeac560abfda3637e103d38763c860197c6b0/x-pack/plugins/triggers_actions_ui/public/application/lib/action_connector_api.ts#L11-L14

Reply:
Did the same in actions client. This one in triggers_actions_ui should be removed in the UI PR